share_app/src/main/java/org/odk/share/fragments/BlankFormsFragment.java

```diff
@@ -95,10 +95,12 @@ public void onLoadFinished(@NonNull Loader<Cursor> loader, Cursor cursor) {
         cursor.moveToFirst();
         formAdapter = new FormsAdapter(getActivity(), cursor, this, selectedForms);
         recyclerView.setAdapter(formAdapter);
+        setEmptyViewVisibility(cursor.getCount());
         if (formAdapter.getItemCount() > 0) {
             toggleButton.setText(getString(R.string.select_all));
             toggleButton.setEnabled(true);
         } else {
+
```

Comment:
Please delete this extra line

Reply:
@shobhitagarwal1612 Thanks for pointing out. Done.